27.md

```diff
@@ -1,22 +1,64 @@
 
-When creating an event, clients should include mentions to other profiles and to other events in the middle of the `.content` using [NIP-21](21.md) codes, such as `nostr:nprofile1qqsw3dy8cpu...6x2argwghx6egsqstvg`.
+# Recognized Formatting
 
-Including [NIP-18](18.md)'s quote tags (`["q", "<event-id> or <event-address>", "<relay-url>", "<pubkey-if-a-regular-event>"]`) for each reference is optional, clients should do it whenever they want the profile being mentioned to be notified of the mention, or when they want the referenced event to recognize their mention as a reply.
+- A `>` at the beginning of a line followed by a single space MAY be used to indicate quotations
+- A `#` at the beginning of a word MAY be used to indicate a topic and SHOULD have a corresponding `t` tag
+- A `$` at the beginning of a word MAY be used to indicate a stock ticker
+- Matched `*` characters within a single line MAY be used to indicate emphasis
+- Matched `:` characters around a single word MAY be used to indicate emojis (see [NIP 30](./30.md))
+- Single-backticks with no language tag MAY be used for inline code blocks
+- Triple-backticks with no language tag MAY be used for multi-line code blocks
+- Unordered lists MUST be one level deep, contain only one line, and SHOULD use the `-` character
+- Ordered lists MUST be one level deep, contain only one line, and MUST be sequentially numbered
 
-A reader client that receives an event with such `nostr:...` mentions in its `.content` can do any desired context augmentation (for example, linking to the profile or showing a preview of the mentioned event contents) it wants in the process. If turning such mentions into links, they could become internal links, [NIP-21](21.md) links or direct links to web clients that will handle these references.
+## Example
 
----
+```
+> This is a multiline quote
+>
+> And it ends here
+
+#bitcoin $BTC *emphasis* :custom_emoji: `inline code`
+
+- This
+- is
+- an
+- unordered
+- list
+
+1. This
+2. is
+3. an
+4. ordered
+5. list
+```
+
+# Embedding References
+
+URLs (excluding base64 encoded image data), [NIP 19](./19.md) nostr entities, prefixed with `nostr:`, and [NIP 73](./73.md) external content ids MAY be embedded in NUML `content`.
+
+Clients MAY display these references as-is, or they MAY use special handling when rendering them. When rendering references:
+
+- Inline references SHOULD be handled differently from references placed on their own line.
+- References on their own line MAY be rendered by embedding a representation of them (an image, video, card, audio player, etc).
+- Domain names without schema or path (e.g. `nytimes.com`) SHOULD be rendered as text.
 
-## Example of a profile mention process
+When publishing content:
+
+- A [NIP 18](./18.md) `q` tag pointing to any `nostr:nevent1...` or `nostr:naddr1...` references MAY be included if filtering by quoted entity is desired.
```

Comment:
Reusing tag names with different formats is very common on Nostr. I do think it is important to redefine their format in every NIP. Otherwise, we won't know which `q` and `p` tag we are talking about.

Reply:
I disagree, I don't see how a reference to NIP 18 leads to ambiguity